docs/public-networks/concepts/node-sync.md

```diff
@@ -108,6 +117,11 @@ You need Besu version 22.4.3 or later to use checkpoint sync.
 Checkpoint sync behaves like [snap sync](#snap-synchronization), but instead of syncing from the
 genesis block, it syncs from a specific checkpoint block configured in the [Besu genesis file](genesis-file.md).
 
+:::info important
+When you run a fresh checkpoint sync, Besu skips all pre-merge PoW blocks and discards
```

Comment:
```suggestion
When you run a fresh checkpoint sync, Besu skips all pre-merge PoW block bodies and
```

Reply:
discards implies it downloads and discards, it skips them the same as the block bodies